File: programmers/level01/2016년.py
# https://programmers.co.kr/learn/courses/30/lessons/12901

def solution(a, b):
    days = [31,29,31,30,31,30,31,31,30,31,30,31]
    day_of_week = ['THU','FRI','SAT','SUN','MON','TUE','WED']
    total_days = b
    for i in range(0, a-1):
        total_days += days[i]
    answer = day_of_week[total_days%7]
    return answer
    
# if __name__ == "__main__":
#     print(solution(5, 24))

File: problem-solving/programmers/level01/2016년.java
// https://programmers.co.kr/learn/courses/30/lessons/12901
import java.util.*;

class Solution {
    public String solution(int a, int b) {
        String answer = "";

        int[] days = {31,29,31,30,31,30,31,31,30,31,30,31};
        String[] weekdays = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        int resultDay = 0;
        for (int i=0; i<a-1; i++) {
            resultDay += days[i];
        }
        resultDay += b-1;
        answer = weekdays[resultDay%7];
        return answer;
    }
}